### PR TITLE
Feature/283 page discoverability

### PR DESCRIPTION
**Pull request by Niekvandam**
## Description

Adding a basic form of SEO to the website, to get a small start on getting discovered by new visitors

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [X] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [X] My code follows the style guidelines of this project
-   [X] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [X] I updated the changelog with an end-user readable description
-   [X] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

### Check sitemap
1. Go to the root website /sitemap.xml
2. You should see an xml document which contains the sitemap

### Check robots.txt
1. Go to the root website /robots.txt
2. You should see a basic robots.txt file 

### Lazy loading images
1. Go to the website
2. Check if all images load correctly

## Link to issue

Closes: #283

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added basic SEO for better google discoverability - [#283](https://github.com/orgs/DigitalExcellence/projects/11#card-45582851)
```
Comment (MaxKorlaar) [inline CHANGELOG.md:12]: Changelog entry should link to issue page, not the issue card in the project overview 😃

File: src/sitemap.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url><loc>http://dex.software/home</loc><priority>0.5</priority></url>
+    <url><loc>http://dex.software/home/project/overview</loc><priority>0.5</priority></url>
+    <url><loc>http://dex.software/home/project/add/source</loc><priority>0.5</priority></url>
+    <url><loc>http://dex.software/home/account/login</loc><priority>0.5</priority></url>
```
Comment (MaxKorlaar) [inline src/sitemap.xml:6]: These should refer to the canonical URLs on DeX, which are all secure (https), these URLs all redirect.

File: src/sitemap.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+    <url><loc>http://dex.software/home/privacy</loc><priority>0.5</priority></url>
```
Comment (MaxKorlaar) [inline src/sitemap.xml:7]:
```suggestion
```
I don't think that the privacy policy is relevant for a sitemap. I leave it out on all my own sites. Search engines also decide what's important as well.

Also, if you were to leave it in, the URL's incorrect.

File: src/sitemap.xml (hunk 2 above)
Comment (MaxKorlaar) [inline src/sitemap.xml:6]: These URLs are all broken

File: src/index.html [hunk 4]
```diff
@@ -25,6 +25,7 @@
   <title>Digital Excellence</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="Dex provides a platform for students to share and work on projects. Share, work on and create new projects on Dex">
```
Comment (MaxKorlaar) [inline src/index.html:28]:
```suggestion
  <meta name="description" content="DeX provides a platform for students, teachers and employees to share and work on projects. Share, work on and create new projects on DeX">
```

**Review by MaxKorlaar — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 4 above)
Comment (Brend-Smits) [reply src/index.html:28]:
```suggestion
  <meta name="description" content="DeX provides a platform for students, teachers and employees to share and work on projects and ideas. Find, create, share and work on projects and ideas on DeX">
```
DeX is about projects AND ideas ;)

**Review by Brend-Smits — CHANGES_REQUESTED** [review]
Welcome and congrats on your first PR 🎉

**Review by juliaslaats — COMMENTED** [review]
Looks good to me (but I'm not sure I understand enough of what you did to realise if anything is wrong)

**Review by MaxKorlaar — APPROVED** [review]

**Review by Brend-Smits — APPROVED** [review]

**Pull request merged**